pymc/sampling.py

```diff
@@ -471,6 +464,20 @@ def sample(
 
     if isinstance(step, list):
         step = CompoundStep(step)
+    elif isinstance(step, NUTS):
```

Comment:
What happens with compound steps that include NUTS? Aren't we initializing things the same way then?

Reply:
We never did, this is related to my comment on slack. I can try to address that, but I preferred to first make this change that revert us to what we "always" did  (plus being able to have external samplers like BART) so we can make a quick release.